### PR TITLE
remove ogc from build pipeline, knowingly breaking release-microk8s-{arch}

### PR DESCRIPTION
**Pull request by addyess**

### CODE REVIEW

File: cilib.sh [hunk 1]
```diff
@@ -7,43 +7,6 @@ if [[ $0 == $BASH_SOURCE ]]; then
 fi
 echo "sourced ${BASH_SOURCE:-$0}"
 
-setup_env()
```
Comment (addyess) [inline cilib.sh:10]: these shell commands didn't seem to be used in this repo
```
setup_env
inject_env
collect_env
wait_env
```

File: jobs/ci-master.yaml [hunk 2]
```diff
@@ -11,18 +11,6 @@
         shallow-clone: true
         clean: {before: true}
 
-- scm:
```
Comment (addyess) [inline jobs/ci-master.yaml:14]: a search through the repo show'd no references to `k8s-jenkins-bm`

File: jobs/ci-master.yaml [hunk 3]
```diff
@@ -57,21 +45,6 @@
           description: force run regardless of condition
 
 
-- parameter:
-    name: ogc-spec-params
```
Comment (addyess) [inline jobs/ci-master.yaml:61]: a search through the repo shows no mention of `ogc-spec-params`

File: jobs/ci-master.yaml [hunk 4]
```diff
@@ -265,21 +238,6 @@
           env > $WORKSPACE/.env
 
 
-- builder:
-    name: ogc-execute-spec
```
Comment (addyess) [inline jobs/ci-master.yaml:269]: a search through the repo shows no use of `ogc-execute-spec`

File: requirements.in [hunk 5]
```diff
@@ -23,9 +23,6 @@ kv
 loguru
 mkdocs-material>=5.3.0
 mkdocs
-ogc-plugins-juju>=1.0.39
-ogc-plugins-runner>=1.0.24
-ogc>=1.0.1
```
Comment (addyess) [inline requirements.in:28]: `ogc-plugins-juju` and `ogc-plugins-runner` were [removed from pypi](https://github.com/adam-stokes/ogc/issues/11) by Mr. Stokes and to mitigate this loss, will purge these from the requirements.

**Review by kwmonroe — APPROVED** [review]
+1; we'd break all jenkins jobs without this.  We'll fix the release-uk8s job in a subsequent PR.

**Pull request merged**